tests/30-interop.js

```diff
@@ -4,17 +4,22 @@
 'use strict';
 
 const chai = require('chai');
-const {implementations} = require('vc-api-test-suite-implementations');
+const {
+  filterByTag,
+  implementations
```

Comment:
I think `implementations` is not being used anymore. If that's the case, this can be removed.